.github/workflows/release-packages.yaml

```diff
@@ -91,7 +91,7 @@ jobs:
       - name: Get release tag name
         # The GITHUB_REF we get has refs/tags/ in front of the tag name so we
         # strip that here
-        run: echo "::set-env name=RELEASE_TAG::${GITHUB_REF/refs\/tags\/}"
+        run: echo "RELEASE_TAG=${GITHUB_REF/refs\/tags\/}" >> $GITHUB_ENV
```

Comment:
I think I have tried to refer to Github environmental variables file through $GITHUB_ENV and I was unsuccessful, does it work for you? I had to use $env:GITHUB_ENV instead of $GITHUB_ENV.

Reply:
Oh, that just depends on whether that's Windows or Unix, I believe. Powershell requires `$env:VARIABLE_NAME`, while bash should work this way.